src/v/kafka/protocol/errors.h

```diff
@@ -220,6 +220,8 @@ enum class error_code : int16_t {
     fenced_instance_id = 82,
     // This record has failed the validation on broker and hence be rejected.
     invalid_record = 87,
+    // Broker declined to process request due to exceeded resource quotas.
+    throttling_quota_exceeded = 89,
```

Comment:
The KIP mentions doing client version checks before sending this new-ish response code  -- do we need to do that?

(although per my comment on cluster::errc::quota_exceeded, I'm not sure if we're actually ever returning this error)

Reply:
Fixed